src/styles.css

```
@@ -80,12 +106,36 @@ body {
+  --mdc-filled-button-container-color: var(--cap-brown);
+  --mdc-filled-button-label-text-color: antiquewhite;
+  --mdc-filled-button-disabled-container-color: var(--border-color);
+  --mdc-filled-button-disabled-label-text-color: var(--highlight-color);
+  --mdc-text-button-label-text-color: var(--text-color);
 }
 
 /* Apply variables to body */
 body {
   background-color: var(--background-color);
   color: var(--text-color);
+  font-family: 'Roboto', "Helvetica Neue", sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)